dot/parachain/prospective-parachains/messages.go

```diff
@@ -0,0 +1,173 @@
+package prospective_parachains
```

Comment:
```suggestion
package prospectiveparachains
```

https://go.dev/blog/package-names

Conventionally, we don't use underscore or camelcase in go package names.

Reply:
Updated